2020-03-04-all-hands-meeting.md

```diff
@@ -0,0 +1,194 @@
+
+### Task Board Review
+
+- Archive _Done_ tasks :tada:
+    - ...
+    - OKR one. pc: is someone leading? bl: #todo (see #199)
+    - #154 quickbooks
+- Review [HIGH `[priority-★★★]`][l-pri-hi]
+    - private#20 QM.
+    - private#18 Free.
+        - had call. more one-off than. ladder maybe needed. convos needed: WSIB, insurance. Do we want to do this sort of thing?
+        - discussion will continue in Infra WG meeting
+        - uv: flagging that if seems complex, client might lose interest. +1 pc
+        - yj: i often am relaxed on meeting these sorts of requirements
+            - pc bl: wouldn't block.
+    - #156 payroll. one final thing to check off.
+    - #212 domains. no block on expiring.
+    - #199 OKR steward. no progress.
+    - #102 AGM. no progress.
+    - #77 brand identity. ongoing.
+        - uv: color scheme. wordmark. some idea of visual imagery. initial brand identity = done.
+        - created new task for swap out color, font, image. also export square logo [`organizing#219`](https://github.com/hyphacoop/organizing/issues/219)
+- Triage new tasks
+- missing labels [`wg:NULL`][l-none] | [`[priority-NULL]`][l-pri-none]
+
+### Working Group Updates
+
+- [Business Planning][biz-wg] :white_check_mark: [`wg:business-planning`][l-biz]
+    - Talked about opportunities: SXSW/starling. Dweb.
+    - Resolved issues to do with intro deck
+- [Finance][fin-wg] :white_check_mark: [`wg:finance`][l-fin]
+    - Writing guides, knowledge xfer from patcon, relating progress to OKRs.
+- [Governance][gov-wg] :white_check_mark: [`wg:governance`][l-gov]
+    - dc not here. Talked about meeting cadence. More aggressive scheduling
+- [Infra & Ops][ops-wg] :white_check_mark: [`wg:infra-ops`][l-ops]
+    - Opportunity with Free
+    - Drawing from OKR's
+    - Find new biz dev opportunities with prospective clients
+    - Emails up and running
+
+### Project Updates
+
+- StART
+    - pc:
+        - We had our combined review/retro/planning, went well though dc was unavailable
+        - First demo with features to show!
+        - Seems c4c felt good even though things aren't perfect
+        - New "definition of done" criteria to ensure all new features have usage tracking
+- Quartermaster
+    - uv:
+        - thurs/fri next week signing. had one paid session, but no contract yet.
+        - pushing invoice cycle out a month. team ok not being paid right away.
+        - had first session. dc and uv came prepped with 20 project write-ups.
+            - used as way to talk about different directions he could take. client really liked this.
+        - project = currently very open-ended and ambiguous.
+        - offering will likely be lots of workshopping.
+        - next up: product walkthrough. show data. share issues.
+- Aether
+    - bl:
+        - now getting user feedback. better idea of market positioning. dev can now be guided by immediate user needs. use of jira board with task priorities bl can pick up.
```

Comment:
@benhylau  as we're using specific tools for StART, I'm adding them to platform inventory. Does Jira belong there as a tool you're using on a project? I have no opinion, just asking :)

Reply:
I use gitlab and jira for aether, but perhaps project specific stuff shouldn't be in our service inventory? not sure

Reply:
Also not sure, or could we ringfence those mention of other tools?

Reply:
Ringfence sounds ok to me (yay new word), but happy to defer. Could also just link project-specific platform inventory docs, since StART has its own. 🤷‍♂